hazelcast/src/main/java/com/hazelcast/client/cache/impl/ClientCacheProxySupportUtil.java

```diff
@@ -55,7 +55,7 @@
 /**
  * Helper class mainly for {@link ClientCacheProxySupport} that contains utility-methods and utility-classes.
  */
-final class ClientCacheProxySupportUtil {
+public final class ClientCacheProxySupportUtil {
```

Comment:
Shouldn't this remain internal?

Reply:
Seems it is better to have it internally. Update PR.